components/x-privacy-manager/src/__tests__/privacy-manager.test.jsx

```diff
@@ -34,18 +34,11 @@ const buildPayload = (consent) => ({
 			},
 		},
 	},
-	cookieDomain: '.ft.com',
 	formOfWordsId: 'privacyCCPA',
 });
 
-function checkPayload(opts, expected) {
-	const consents = JSON.parse(String(opts.body)).data;
-
-	let worked = true;
-	for (const category in consents) {
-		worked = worked && consents[category].onsite.status === expected;
-	}
-	return worked;
+function getLastFetchPayload() {
+	return JSON.parse(fetchMock.lastOptions().body);
```

Comment:
With this change the fetch payload is now check for deep-equality instead of just checking the category status flags. In my opinion it's more straight-forward and relies on less code